stages/org.osbuild.oscap.autotailor.meta.json

```diff
@@ -9,17 +9,27 @@
     " - requires `openscap-utils` package in the buildroot."
   ],
   "schema": {
-    "additionalProperties": false,
-    "required": [
-      "filepath",
-      "config"
-    ],
-    "properties": {
-      "filepath": {
-        "type": "string",
-        "description": "Filename and path to where the tailoring file will be saved"
+    "definitions": {
+      "json-tailoring": {
+        "additionalProperties": false,
+        "required": [
+          "datastream",
+          "json_tailoring_filepath"
+        ],
+        "type": "object",
+        "description": "OpenSCAP tailoring configuration variables",
+        "properties": {
+          "datastream": {
+            "type": "string",
+            "description": "The path to the datastream file"
+          },
+          "json_tailoring_filepath": {
```

Comment:
Could this simply be `filepath` since it's already under `json-tailoring` or are we afraid it'll be confusing with the other `filepath` in the schema?

Reply:
I actually think `tailoring_filepath` (or maybe `tailoring_file`) would be better than `json_filepath`.  I agree that having another plain `filepath` property here might be confusing, but I think a property name describing its purpose (tailoring) is more useful than a name describing the format (json).

Reply:
Makes sense. It's in the `json-tailoring` block so `tailoring_file` is reasonable. Also, I updated the description to make this a bit clearer too